### ISSUE TITLE
Decide how to handle admin set setup

### ISSUE THREAD

**Issue by mark-dce** [curationexperts/laevigata#79]
**ISSUE**
At present you need to run a rake task for each new environment.  We either need to add this to all set up instructions, or automate the setup of admin sets so that it can be handled via the UI

- [x] Make a system to automate the setup of admin sets
- [x] Ensure it is run whenever the application is deployed
- [x] What admin sets and workflows and user identities do we want automatically set up? 
  - [x] 4 admin sets, one for each school, just one workflow for them all, everything owned by uberadmin@localhost.com

**Comment by bess** [comment]
An admin set is tied to a workflow, and the creation and use of workflows is a little complicated. I don't think we want to do this via the UI. I think it has to be automated and needs to stick with the application, more like a database migration. Also, it's a prerequisite for having an approval workflow, so I'm moving it up on the priority list.

**Comment by mark-dce** [comment]
Agreed, the UI part of the description might have been a red-herring from trying to transcribe the discussion as it was happening.

There are other types of seed information that we've needed in various past projects.  One of the patterns we've used is to check the seed configuration into the repo, have some form of initializer check for the existence of the the type of configuration expected (eg. is there more than one admin set - we probably expect one for each school) and if the check fails to load the seed data into the environment in question.

As an example use case:  I've configured an admin set per school, I run rake solr_clean + rake fc_repo clean to clear out test data I've been messing around with.  I still want my admin sets configured in a consistent manner without having to run multiple rake tasks.  I also want everyone else's dev, test, and production environments to have the same predictable configuration...
